server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -242,8 +255,10 @@ public Invocations getInvocations() {
 
       ConfigurationBuilder finalCfgBuilder = cfgBuilder;
       return CompletableFuture.supplyAsync(() -> {
-         administration.createCache(cacheName, finalCfgBuilder.build());
-
+         Security.doAs(request.getSubject(), (PrivilegedAction<Void>) () -> {
```

Comment:
Same

Reply:
i thought I modified those already ... hmmm
doing the changes